modules/datadict/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary.
+
+## Scope
+
+The module displays and manages the data dictionary for things
+already stored in the LORIS `parameter_type` table. This table is
+generally autopopulated and provides a dictionary for imaging headers
+(populated by the imaging pipeline) and behavioural instrument
+(populated by the `lorisform_parser.php` script).
+
+It does not provide a way to enter new things into the data dictionary,
+or describe the data dictionary for arbitrary SQL tables which are
```

Comment:
SQL -> MySQL

Reply:
This module doesn't do anything MySQL specific. It would work just as well with any other SQL engine if we ever decided to support it.